site/install.md

```diff
@@ -69,6 +69,15 @@ in `requirements.txt` to install the latest patch release.
 These include the Apache Arrow and Apache Parquet C++ binary libraries bundled
 with the wheel.
 
+### C++ Library on Homebrew (macOS)
```

Comment:
Could you add "(Unofficial)" mark like we did for conda packages above?
We should say "unofficial" explicitly for packages unvoted by Apache Arrow PMC.

Reply:
Could you also add "and GLib (C)"?
We can install Apache Arrow GLib as apache-arrow-glib formula: https://formulae.brew.sh/formula/apache-arrow-glib

Reply:
Done, along with the other requested changes, PTAL.

Why are these "unofficial"? Aren't they built from the 0.13.0 release that was voted on? How are the Python wheels "official" and these aren't? How can we make all of our packages official?

I get that this may be an important distinction within the Apache community, but to people who aren't familiar with what that means, it sounds like we're raising doubts about the quality of our own software.

Reply:
@nealrichardson it's an Apache process thing. We can't call a package "official" unless 

* It is a source package voted on by the PMC
* It is a binary artifact voted on by the PMC
* It is a built locally from an official source tarball

If the Homebrew packages are built locally from the official source tarball then I don't think they need to be marked unofficial. I think HB caches some binaries of things so if those are being installed then we can't call them official

On the installation page we should add a section to clarify what is meant by "unofficial"

For the record nearly all Spark and Hadoop-related builds used in production are produced by vendors and are not official ASF releases, so this is already the norm

Reply:
FWIW the words "official" and "unofficial" don't appear on either http://spark.apache.org/downloads.html or https://hadoop.apache.org/releases.html. https://wiki.apache.org/hadoop/Distributions%20and%20Commercial%20Support?action=show&redirect=Distribution has a couple of sentences about what defines "a release of Apache Hadoop" that might be a good reference for us.

Reply:
Right, Spark and Hadoop are pretty different projects in terms of how they are installed and deployed in production compared with our C++ and Python binaries. It's a semantic / language issue but we are unable to call any artifact that we haven't voted on a "release", so we either have to have a clear delineation between official/unofficial or not list the not-voted-on artifacts at all

Reply:
Normally, Homebrew users install Homebrew formulas as "bottle" rather than build locally from source. "Bottle" is a binary package: https://docs.brew.sh/Bottles

If "bottle" is available for the formula, Homebrew uses "bottle" by default. So I think that "unofficial" is suitable for Homebrew.